app/presenters/publishing_api_presenters/edition.rb

```diff
@@ -37,10 +37,6 @@ def as_json
 
   private
 
-    # An incomplete and temporary details hash that only currently includes
-    # specialist topics. This is to enable the email alerts service to generate
-    # email alerts based on taggings to topics.
-    #
     # Note that we are referencing tags using the temporary `tags` hash here in
```

Comment:
This paragraph should probably be updated to also mention policies as well as topics.